### PR TITLE
support wjh module config with testing file

### PR DESCRIPTION
**Pull request by anas-shami**
##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/network/onyx/onyx_wjh.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: onyx_wjh
+version_added: "2.5"
```
Comment (samerd) [inline lib/ansible/modules/network/onyx/onyx_wjh.py]: 2.9

File: lib/ansible/modules/network/onyx/onyx_wjh.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+notes:
+options:
+    group:
+        description: Name of wjh group
+        required: true
+        choice: ['all', 'forwarding', 'acl']
+    enabled:
+        description: wjh group status
+        type: bool
+    auto-export:
```
Comment (samerd) [inline lib/ansible/modules/network/onyx/onyx_wjh.py]: auto_expert

File: lib/ansible/modules/network/onyx/onyx_wjh.py [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        if config_lines:
+            self._set_current_config(config_lines)
+
+    '''
+        wjh is enabled "by default"
+        when wjh disable we  won't find wjh commands in running config 
+    ''' 
+    def generate_commands(self):
+        current_config, required_config = self._current_config, self._required_config
+        for key, value in required_config.items():
```
Comment (samerd) [inline lib/ansible/modules/network/onyx/onyx_wjh.py]: Are the keys mutual exclusive?

**Review by samerd — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/onyx/onyx_wjh.py (hunk 3 above)
Comment (anas-shami) [reply lib/ansible/modules/network/onyx/onyx_wjh.py]: yes I just connect between the value of current config and use keys to indicate what is the required config.

**Pull request closed without merging**